backtrace-library/src/main/java/backtraceio/library/services/BacktraceReportSender.java

```diff
@@ -102,6 +112,80 @@ static BacktraceResult sendReport(String serverUrl, String json, List<String> at
         return result;
     }
 
+    /**
+     * Send HTTP request for certain url server with information about events
+     *
+     * @param serverUrl             server http address to which the request will be sent
+     * @param json                  message wih information about events
+     * @param payload               information about events
+     * @param errorCallback         event that will be executed after receiving an error from the server
+     * @return information from the server about the result of processing the request
+     */
+    public static EventsResult sendEvents(String serverUrl, String json, EventsPayload payload, OnServerErrorEventListener errorCallback) {
+        HttpURLConnection urlConnection = null;
+        EventsResult result;
+        int statusCode = -1;
+
+        try {
+            URL url = new URL(serverUrl);
+            urlConnection = (HttpURLConnection) url.openConnection();
+
+            urlConnection.setRequestMethod("POST");
+            urlConnection.setUseCaches(false);
+
+            urlConnection.setDoOutput(true);
+            urlConnection.setDoInput(true);
+
+            urlConnection.setChunkedStreamingMode(128 * 1024);
+            urlConnection.setRequestProperty("Connection", "Keep-Alive");
+            urlConnection.setRequestProperty("Cache-Control", "no-cache");
+
+            urlConnection.setRequestProperty("Content-Type", RequestHelper.getContentType());
+
+            BacktraceLogger.d(LOG_TAG, "HttpURLConnection successfully initialized");
+            DataOutputStream request = new DataOutputStream(urlConnection.getOutputStream());
+
+            RequestHelper.addJson(request, json);
+            RequestHelper.addEndOfRequest(request);
+
+            request.flush();
+            request.close();
+
+            statusCode = urlConnection.getResponseCode();
+            BacktraceLogger.d(LOG_TAG, "Received response status from Backtrace API for HTTP request is: " + Integer.toString(statusCode));
+
+            if (statusCode == HttpURLConnection.HTTP_OK) {
+                result = BacktraceSerializeHelper.fromJson(getResponse(urlConnection), EventsResult.class);
```

Comment:
do we need at all deserialization? What do we get by deserializing output?

Reply:
I don't think we need this, thank you!